### ISSUE TITLE
Android BackHandler issue when video is in fullScreen mode.

### ISSUE THREAD

**Issue by ppv94** [inProgress-team/react-native-youtube#418]
I am using react-native-youtube in android when we play video in full screen and click the android back button to minimise the video whole application orientation changes to landscape mode, how to resolve this issue. 

`"react-native": "0.61.2",
"react-native-youtube": "^2.0.0",`

**Comment by jothikannan** [comment]
Yes, I am also having the same issue, please fix it as soon as possible? @ppv94 is video is playing on inline mode? For me, video is playing for1sec and stopping automatically

**Comment by davidohayon669** [comment]
@ppv94, @jothikannan can you offer a PR?

**Comment by samuelchow1997** [comment]
> Yes, I am also having the same issue, please fix it as soon as possible? @ppv94 is video is playing on inline mode? For me, video is playing for1sec and stopping automatically

same here.

**Comment by ebrahimmelngary** [comment]
I have same issue, try this slouation
https://reactnavigation.org/docs/en/navigation-events.html#component-props
